src/evidently/ui/utils.py

```diff
@@ -35,9 +35,10 @@ def _request(
             method, urllib.parse.urljoin(self.base_url, path), params=query_params, data=data, headers=headers
         )
         response.raise_for_status()
+        response_json = response.json()
```

Comment:
this will fail if response content is not json. if you want raw dict returned, just use `dict` as response_model

Reply:
or even better if you use actual model of the response :)

Reply:
If a generic client is really needed, then it should contain only the logic for executing the request and handling retryable exceptions, while the response processing should be implemented on the side of the specific client. The current implementation, when fully working with mypy, will only cause problems, as the function returns either T or Response. I will refactor the code, but the client needs a redesign